solutions/fscloud/version.tf

```diff
@@ -1,10 +1,9 @@
 terraform {
   required_version = ">= 1.3.0, <1.6.0"
   required_providers {
-    # Use latest version of provider in non-basic examples to verify latest version works with module
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = ">= 1.56.1"
+      version = "1.56.1"
```

Comment:
Can you update this to 1.62.0 since its the latest and renovate will always use bump this to latest since its a DA